pkg/vulnsrc/osv/testdata/happy/vuln-list/osv/python/withdrawn_after_now.json

```diff
@@ -0,0 +1,102 @@
+{
+  "id": "PYSEC-2019-156",
+  "modified": "2021-07-05T00:01:27.093286Z",
+  "published": "2019-11-04T20:15:00Z",
+  "withdrawn": "2024-06-06T10:37:00Z",
```

Comment:
This test will fail next year. Can we change it 3000 or something?

Reply:
let's wait for it.
I'm going to see it. while everything is going well ))